_data/implementations.yml

```diff
@@ -1,3 +1,19 @@
+- name: Sam-Prototypes
+  distribution: Angular
+  url: https://github.com/GSA/sam-prototypes
+  author:
+    name: SAM.gov
+    url: https://github.com/GSA
+  notes: "A test application that incorporates sam-styles and sam-design-system components with real-world examples"
+
+- name: Sam-Design-System
+  distribution: Angular
+  url: https://github.com/GSA/sam-design-system
+  author:
+    name: SAM.gov
+    url: https://github.com/GSA
```

Comment:
I was thrown off by the the `SAM.gov` link pointing to GSA's GitHub. Does it make sense to but the author name as GSA? Or maybe something like SAM.gov/GSA?

Reply:
Probably GSA if it's from there.